packages/services/api/src/modules/organization/module.graphql.ts

```diff
@@ -240,40 +244,54 @@ export default gql`
   }
 
   input InviteToOrganizationByEmailInput {
-    organizationSlug: String!
-    roleId: ID
-    email: String!
+    organization: OrganizationReferenceInput! @tag(name: "public")
+    email: String! @tag(name: "public")
+    memberRoleId: ID @tag(name: "public")
   }
 
   input DeleteOrganizationInvitationInput {
-    organizationSlug: String!
-    email: String!
+    organization: OrganizationReferenceInput! @tag(name: "public")
+    email: String! @tag(name: "public")
   }
 
-  type InviteToOrganizationByEmailError implements Error {
-    message: String!
+  type InviteToOrganizationByEmailResultError {
+    message: String! @tag(name: "public")
     """
     The detailed validation error messages for the input fields.
     """
-    inputErrors: InviteToOrganizationByEmailInputErrors!
+    inputErrors: InviteToOrganizationByEmailInputErrors! @tag(name: "public")
+  }
+
+  type InviteToOrganizationByEmailResultOk {
+    createdOrganizationInvitation: OrganizationInvitation! @tag(name: "public")
   }
 
   type InviteToOrganizationByEmailInputErrors {
-    email: String
+    email: String @tag(name: "public") @tag(name: "public")
   }
 
   """
   @oneOf
   """
   type InviteToOrganizationByEmailResult {
-    ok: OrganizationInvitation
-    error: InviteToOrganizationByEmailError
+    ok: InviteToOrganizationByEmailResultOk @tag(name: "public")
```

Comment:
This is a breaking change. I doubt these are used by anyone else, but we should check before deploying.